.github/workflows/ci.yml

```diff
@@ -0,0 +1,187 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [published]
+  schedule:
+    # Daily at 3:21
+    - cron: '21 3 * * *'
```

Comment:
I dunno about this section.  There's no periodic CI on any other platforms.  Maybe it's a good thing but I think it warrants some additional consideration.  Let's leave it out for now.

Reply:
Dropped `schedule` in 9752d6a.  I was curious to see how this works, but it never did on my branch while I was working on the PR.

Would you like to drop  `release` as well?  I merely copy-pasted both `schedule` and `release` from the example linked from https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3277, and I have not seen either of them working.

Reply:
Hm.  I'm not even really sure what "release" does here.  Build on creation of GitHub release objects, I guess?  I guess the release process involves creating those.  Maybe it's better to leave this out until we better understand it, though (and avoid having that understanding block the rest of this).

Reply:
[Documentation](https://help.github.com/en/actions/reference/events-that-trigger-workflows#release-event-release) says that `on.release` will run "your workflow anytime the release event occurs."

It probably makes sense to use this on an as-needed basis.  Removed in a38029f.